views/default/default/getAdvice.ejs

```diff
@@ -78,7 +78,7 @@
                  <li> Your data publication can be cited as readily as your traditional publications, because we can publish it with the required citation, including a DOI; </li>
                  <li> Your data may be important to the future of knowledge in unpredictable ways; </li>
                  <li> Open data is a crucial element of scientific enquiry;</li>
-                 <li> It is required that you <em> consider </em> data publication under the <a href="http://www.gsu.uts.edu.au/policies/research-management-policy.html" target="_blank"> UTS Research Management Policy </a> with the understanding that there may be legal, ethical or commercial impediments to doing so.</li>
+                 <li> It is required that you <em> consider </em> data publication under the <a href="https://gsu.uts.edu.au/policies/research.html" target="_blank"> UTS Research Management Policy </a> with the understanding that there may be legal, ethical or commercial impediments to doing so.</li>
```

Comment:
I think this has been split out from just a research policy now, so should be:
```suggestion
                 <li> It is required that you <em> consider </em> data publication under the <a href="https://gsu.uts.edu.au/policies/research.html" target="_blank"> UTS Research Data Management Procedures </a> with the understanding that there may be legal, ethical or commercial impediments to doing so.</li>
```
(see 4.30 of https://gsu.uts.edu.au/policies/documents/research-data-management-procedures.pdf)